lib/Sparql/Client.php

```diff
@@ -392,7 +393,8 @@ protected function parseResponseToQuery($response)
             $result = new Result($response->getBody(), $content_type);
             return $result;
         } else {
-            $result = new Graph($this->queryUri, $response->getBody(), $content_type);
+            $graphClass = TypeMapper::getDefaultGraphClass();
+            $result = new $graphClass($this->queryUri, $response->getBody(), $content_type);
```

Comment:
Why not use `return new $graphClass($this->queryUri, $response->getBody(), $content_type);`? Saves one line of code.